### DISCUSSION TITLE
[Question]: Upload Privacy

### DISCUSSION THREAD

**Discussion by tristandevs** [danny-avila/LibreChat#3504] [Q&A]
### What is your question?

Does the uploaded image on LibreChat’s CDN for vision models expire after a certain period or does it ever get deleted when the user deletes conversations?

### More Details

N/A

### What is the main subject of your question?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There are currently no retention settings but is planned. In general, files only ever expire if they were never submitted when attached

**Reply by danny-avila** [reply]
https://github.com/danny-avila/LibreChat/issues/2365
